src/main/kotlin/com/terraformation/backend/accelerator/api/ActivitiesController.kt

```diff
@@ -125,6 +126,7 @@ class ActivitiesController(
             activityId,
             file.inputStream,
             FileMetadata.of(contentType, filename, file.size),
+            listPositionStr?.toIntOrNull(),
```

Comment:
Will this throw the correct error if `listPosition` is passed as "ABC" instead of a parseable number?

Reply:
Good point. It ignores invalid numbers right now but it'd be better to reject them.